src/gui/migrations/0026_remove_feedback_fields.py

```diff
@@ -0,0 +1,70 @@
+# Generated by Django 4.2.2 on 2023-07-11 09:06
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("gui", "0025_auto_20220803_0838"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
We should make sure that we're not accidentally deleting data that may still be required.